.travis.yml

```diff
@@ -7,4 +10,6 @@ before_install:
         - brew upgrade carthage || true
         - gem install cocoapods --pre || true
         - export LC_ALL="en_US.UTF-8"
-script: rake test
+script:
+  - carthage update --no-use-binaries
```

Comment:
the `--no-use-binaries` is a fix for hitting the GitHub API limit all the time. And yeah, I'll update that to bootstrap